codebuild/release/prod-release.yml

```diff
@@ -0,0 +1,28 @@
+version: 0.2
+
+env:
+  variables:
+    BRANCH: "master"
+  secrets-manager:
+    TWINE_USERNAME: PyPiAdmin:username 
+    TWINE_PASSWORD: PyPiAdmin:password
+
+phases:
+  install:
+    runtime-versions:
+      python: latest
+  build:
+    commands:
+      - pip install tox
+      - git checkout $BRANCH
+      - tox -e park
+      - tox -e release
+      - git clone https://github.com/aws-samples/busy-engineers-document-bucket.git
+      - cd busy-engineers-document-bucket/exercises/python/encryption-context-complete
+      - tox -e test
```

Comment:
Will this definitely pull the dependency from pypi rather than resolving it locally?

Reply:
Yep, we would have to jump through some hoops to try to resolve the local thing.